Update(Test-Doc): Adding more documentation

phillsatellite opened the pull request:
Added more documentation to the Uplink Checklist and fixed typos from my last pr

hyj1204 commented:
The FAB is updated.
![image](https://user-images.githubusercontent.com/14248245/166788336-1400628f-add7-401c-b92b-5e9dbac02943.png)

phillsatellite committed:
Update(Test-Doc): Adding more documentation 

Added more documentation to the Uplink Checklist and fixed typos from my last pr

lgmarchi commented on ui/docs/testing/uplink-checklist.md:
```diff
@@ -37,6 +37,17 @@ This document provides a checklist top-to-bottom of every step you should be abl
+**Files**
+- [ ] User should be able to Drag+Drop 1 or more files.
+- [ ] User should be able to Drag+Drop Files into a specific Folder.
+- [ ] User should not be able to have 2 Folders with the same name.
```
Phill, here I think it is nice to say "same name inside same folder". Because we can have files with same name in different folders.

lgmarchi commented on ui/docs/testing/uplink-checklist.md:
```diff
@@ -37,6 +37,17 @@ This document provides a checklist top-to-bottom of every step you should be abl
+- [ ] User should be able to Drag+Drop 1 or more files.
+- [ ] User should be able to Drag+Drop Files into a specific Folder.
+- [ ] User should not be able to have 2 Folders with the same name.
+- [ ] User should not be able to upload 2 files with the same name.
```
Same above here.

lgmarchi commented on ui/docs/testing/uplink-checklist.md:
```diff
@@ -64,26 +75,49 @@ This document provides a checklist top-to-bottom of every step you should be abl
+- [ ] Folder should highlight when User is drag+dropping a file into it.
+- [ ] Folder should also be highlighted when User hovers cursor over it.
+- [ ] Upload % should show when User is uploading Files 
+- [ ] Clicking the _Home_ buttin in Directory should take you to Files Home page.
```
I think `buttin` is `button` here.

lgmarchi reviewed (approved):
Just did small comments to change some words or add some words in description. 

Nice checklist!

lgmarchi commented on ui/docs/testing/uplink-checklist.md:
```diff
@@ -64,26 +75,49 @@ This document provides a checklist top-to-bottom of every step you should be abl
+- [ ] Size of file should show underneath preview. 
+- [ ] Amount of items/size of uploaded files should show underneath the Folder. 
+- [ ] Directory should be highlighted when User hovers cursor over it.
+- [ ] Upload modal should show path of saif file when User is drag+dropping a file into Modal. 
```
I think `saif` word here should be `said`.

phillsatellite committed:
Update uplink-checklist.md

phillsatellite commented:
@lgmarchi Thanks for taking a look! 😄  I agree with everything above and made the changes 👍
